contrib/mercurial/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "mercurial"
+pkgver = "6.7.3"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+makedepends = [
+    "gettext-devel",
```

Comment:
pretty sure you don't need gettext-devel, and you never need musl-devel, and i'm not sure about the others considering according to the ci output they don't even get linked

Reply:
i was building the rust/C client components but i removed them before submitting the port, forgot to cleanup the deps too